migration/i8.1z/oracle/202011241451_IDEMPIERE-4566.sql

```diff
@@ -0,0 +1,387 @@
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+-- IDEMPIERE-4566 Translation Import/Export - output message is not visible
+-- Nov 24, 2020, 2:45:45 PM CET
+UPDATE AD_Form SET Name='Initial Client Setup Form (deprecated)',Updated=TO_DATE('2020-11-24 14:45:45','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Form_ID=102
```

Comment:
shouldn't the form is mark inactive as well ?

Reply:
Hi @hengsin - the form is already inactive, I was just changing the name to make explicit that is not used anymore.